### PR TITLE
The task should not be Selfservice to be able to reassign.

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
## Issue & Reproduction Steps
the steps are in the ticket
the problem is that the task in selfservice can be reassigned

## Solution
- Add validation in seccion reassign

## How to Test
create process selfservice
create a request
the task in selfservice should not be able to be reassigned

https://user-images.githubusercontent.com/1747025/229132470-de6f2ecc-f631-404f-91d5-e82a211790ef.mp4

## Related Tickets & Packages
- [FOUR-7150](https://processmaker.atlassian.net/browse/FOUR-7150)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

[FOUR-7150]: https://processmaker.atlassian.net/browse/FOUR-7150?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-098530643f.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-098530643f.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-098530643f.eng.processmaker.net

File: resources/views/tasks/edit.blade.php [hunk 1]
```diff
@@ -222,6 +222,7 @@ class="d-inline-flex pull-left align-items-center"
     };
 
     const task = @json($task);
+    console.log(task);
```
Comment (paulosakamoto) [inline resources/views/tasks/edit.blade.php:225]: @marcoAntonioNina ,
This console.log it's important? It's possible to remove it?

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-098530643f.eng.processmaker.net

**Review by boliviacoca — APPROVED** [review]

File: resources/views/tasks/edit.blade.php (hunk 1 above)
Comment (marcoAntonioNina) [reply resources/views/tasks/edit.blade.php]: fixedthis console.log was removed

**Review by paulosakamoto — APPROVED** [review]

**Pull request merged**